### DISCUSSION TITLE
Option to disable "syntax disabled" warnings (Treesitter)

### DISCUSSION THREAD

**Discussion by star-szr** [ibhagwan/fzf-lua#1157] [Ideas]
Hi! Thank you for all you do.

I'm on fzf-lua f7d3023, when using the files picker I keep seeing warnings with the format `syntax disabled for '%s' (%s), consider increasing '%s(%d)'`. It's particularly painful in my case because the first file that opens when I open the files picker is a large file (package-lock.json).

I have my own logic in my treesitter plugin config to disable syntax highlighting for large files, either way I appreciate fzf-lua helping here and would like it to disable the highlighting but would just like it to be quiet about it :)

Thanks!

**Comment by ibhagwan** [comment]
https://github.com/ibhagwan/fzf-lua/commit/2674675a7719e3fb2829748276d9a0b799fd2c7a

@star-szr, makes sense to disable these by default, if someone is missing it they can use `silent=false` :-)

**Reply by star-szr** [reply]
Thank you!
